src/contact.html

```diff
@@ -0,0 +1,50 @@
+---
+layout: default.html
+pageTitle: Contact Us
+tags: nav
+order: 3
+---
+
+<h1 class="title">
+    Contact Us
+</h1>
+
+<section>
+    <h1 class="title">
+        Make an Appointment
+    </h1>
+    <h2>
+        Open Hours
+    </h2>
+    <h3>
+        Select a Date:
+    </h3>
+    <form class="calendar" action="">
```

Comment:
Note that while this meets the requirements of three inputs on a form, the result isn't a functional form -- not in terms of data being saved (I didn't expect anyone to do that), but in that the form doesn't actually ask all the necessary info for it to be useful / relevant to either the museum or the user.

At the bare minimum, this would also need a name field and either an email address or phone number field for it to be useful.